normits_demand/models/notem/production_models.py

```diff
@@ -243,60 +252,52 @@ def run(self,
         # Initialise timing
         # TODO(BT): Properly integrate logging
         start_time = timing.current_milli_time()
-        du.print_w_toggle(
-            "Starting HB Production Model at: %s" % timing.get_datetime(),
-            verbose=verbose
-        )
+        self._logger.info("Starting a new run of NoTEM")
```

Comment:
We don't need this - it's written during the `__init__` function

Reply:
Updated